src/tracks/Pattern.cpp

```diff
@@ -1037,16 +1037,13 @@ void PatternView::paintEvent( QPaintEvent * )
 			{
 				const int vol = n->getVolume();
 				p.drawPixmap( x, y, stepoffl );
+				p.drawPixmap( x, y, stepon0 );
 				if( vol > 0 )
```

Comment:
And this if statement is not really needed, I mean when volume is 0 the opacity will be also 0, effectively rendering nothing.